twofactor_privacyidea/js/push.js

```diff
@@ -0,0 +1,14 @@
+window.onload=function(){
+    setInterval(function() {
+        $.ajax({
+            url:location.href,
+            type:'post',
+            data: $('#piLoginForm').serialize(),
```

Comment:
Please do not send the form data. As we checked yesterday:
If a user has a push token and an OTP token, the failcounter of the OTP token will max out, while the system is waiting for a response from the push token.

Reply:
Fixed